package/MDAnalysis/analysis/distances.py

```diff
@@ -40,13 +40,13 @@
 
 from __future__ import absolute_import
 
-__all__ = ['distance_array', 'self_distance_array',
-           'contact_matrix', 'dist', 'between']
+__all__ = ['contact_matrix', 'capped_distance'
+           'dist', 'between']
```

Comment:
So the history here is that `self_distance_array` and `distance_array` used to live in this module, so when we moved them we had to maintain the API (else we get bugs like you found where `analysis.contacts` imported from here.).

So because this might break someones code if they're doing `from analysis.distances import distance_array` this change needs reverting.